### PR TITLE
0.4.0

### PR DESCRIPTION
**Pull request by sr229**
- [ ] Complete Localizations

### CODE REVIEW

File: src/commands/mari/itsjoke.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-/* 
- * itsjoke.js - It's Joke!.
- *
- * Contributed by Capuccino and Ovyerus.
+/**
+ * @file It's joke! Like me.
```
Comment (sr229) [inline src/commands/mari/itsjoke.js]: Me too thanks

Comment (Ovyerus) [reply src/commands/mari/itsjoke.js]: me too thanks

Comment (sr229) [reply src/commands/mari/itsjoke.js]: sknaht oot em

**Comment by Greek** [comment]
greek translations

**Comment by Greek** [comment]
FUCK

**Comment by Greek** [comment]
sorry

**Comment by sr229** [comment]
but why

**Comment by Greek** [comment]
:feelsgood: 
huh

File: src/cmd/misc/saucenao/saucenao.js [hunk 2]
```diff
@@ -21,37 +21,43 @@ exports.commands = [
 exports.source = {
     desc: 'Tries to find the source for an image.',
```
Comment (sr229) [inline src/cmd/misc/saucenao/saucenao.js]: you might wanna sync this to 0.3.x branch too

File: .eslintrc.json [hunk 3]
```diff
@@ -1,38 +1,36 @@
...
-    ],
-
     "rules": {
         "no-console": 0,
         "global-require":"off",
         
         "quotes": ["warn", "single", {"avoidEscape": true, "allowTemplateLiterals": true}],
         "semi": "warn",
-        "indent": "error",
+        "indent": ["error", 4, {"SwitchCase": 1}],
```
Comment (sr229) [inline .eslintrc.json:32]: Sync codestyle to LTS, it has to apply repo-wide

**Comment by sr229** [comment]
PR still fails Codacy review.

**Comment by sr229** [comment]
Merged.

**Pull request closed without merging**